File: C/Ex19.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){

  float A, V0, T, V;

  printf("Digite o valor de aceleracao: ");
  scanf("%f", &A);
  
  printf("Digite a velocidade inicial ");
  scanf("%f", &V0);

  printf("Digite o valor de tempo de percurso ");
  scanf("%f", &T);

  V = V0 + (A * T);
  
  if(V <= 40){
      printf("Veículo muito lento.");
    }  
    else if(V>40 && V<=60) {
      printf("Velocidade permitida.");
    }
    else if(V>60 && V<=80){
      printf("Velocidade de cruzeiro.");
    } 
    else if(V>80 && V<=120){
      printf("Veículo rápido.");
    }  
    else if(V>120){
      printf("Veículo muito rápido.");
    }
    else{
      printf("Digite um valor válido.");
    }  
  }

File: Python/Ex19.py
aceleracao = float(input("Digite o valor de aceleracao: "))
v0 = float(input("Digite a velocidade inicial: "))
T = float(input("Digite o valor de tempo do percurso: "))
V = v0 + (aceleracao *T)
 
if (V <= 40):
            print("Veículo muito lento.") 

elif(V <= 60):
    print("Velocidade permitida.")

elif(V <= 80):
    print("Velocidade de cruzeiro.")

elif(V <= 120):
    print("Veículo rápido.")

elif(V > 120):
    print("Veículo muito rápido.")

else:
    print("Digite um valor válido. ")